coffee/chosen.proto.coffee

```diff
@@ -504,3 +504,14 @@ class @Chosen extends AbstractChosen
         w = f_width - 10
 
       @search_field.setStyle({'width': w + 'px'})
+
+  trigger_form_field_change:  ->
+    Chosen.trigger(@form_field, 'input')
+    Chosen.trigger(@form_field, 'change')
+
+  @trigger: (element, eventType) ->
+    if (document.createEventObject) # Old IE:
+      element.fireEvent("on#{eventType}", document.createEventObject());
+    else # Modern way:
```

Comment:
you should detect support for the modern way instead (detecting ``dispatchEvent``), to use the modern way when multiple ways are available. ``fireEvent`` should be a fallback only.

Btw, you should also write the code using ``createEvent()`` and ``initEvent`` when ``dispatchEvent`` is available but not the Event constructor (i.e. the intermediate API)